### PR TITLE
[#90742128][1] Dynamic background definition

### PR DESCRIPTION
**Pull request by ingedmundo**
As a developer I want a CSS class for each CrowdInt pantone so that I can get the correct one on my CrowdInt's sites project.

-Creates classes for background and font color, based on the crowdint-styles palette (according lo styleguide provided)

https://www.pivotaltracker.com/n/projects/1193518

### CODE REVIEW

**Comment by zalapeach** [comment]
LGTM

**Comment by jparbros** [comment]
Please, improve your PR description

File: assets/stylesheets/colors.scss [hunk 1]
```diff
@@ -14,15 +13,24 @@ $oracle:          #377267;
...
+              silver-chalice $silver-chalice,
+              mine-shaft $mine-shaft,
+              punch $punch,
+              tango $tango,
+              fun-blue $fun-blue,
+              oracle $oracle,
+              black-haze $black-haze,
+              dust-gray $dust-gray;
+
+@each $context in $colors-list {
```
Comment (jpfuentes2) [inline assets/stylesheets/colors.scss]: Are we creating the classes so other apps which don't have access to the `.scss` files and are only using the compiled `.css` can use the right colors/styles via class names?

Comment (ingedmundo) [inline assets/stylesheets/colors.scss]: Yes, we are creating clases for color and background based on this variables, so other apps without sass can use them.

Comment (jpfuentes2) [inline assets/stylesheets/colors.scss]: :+1:

**Comment by anymoto** [comment]
LGTM

**Pull request merged**